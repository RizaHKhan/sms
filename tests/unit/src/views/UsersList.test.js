import UsersList from '@/views/UsersList.vue';

import testHelpers from '@@/testHelpers.js';

describe('UsersList.vue', () => {
  let localVue;
  let store;
  let setup;

  let permissions = ['student'];

  beforeEach(() => {
    setup = testHelpers.setupTestVariables(permissions);
    localVue = setup.localVue;
    store = setup.store;
  });

  test('Renders correctly', async () => {
    const wrapper = await testHelpers.shallowMountWrapper(UsersList, {
      localVue,
      store
    });

    expect(wrapper)
      .toMatchSnapshot();
  });
});
