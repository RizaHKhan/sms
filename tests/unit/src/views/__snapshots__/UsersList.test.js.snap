// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersList.vue Renders correctly 1`] = `
<div>
  <h1>Users</h1>
  <base-table-stub columndata="[object Object],[object Object]" rowdata></base-table-stub>
</div>
`;
